integration_test.go

```diff
@@ -41,9 +41,9 @@ func testCartographerMap(t *testing.T, svc slam.Service) {
 	test.That(t, err, test.ShouldBeNil)
 	test.That(t, pcd, test.ShouldNotBeNil)
 
-	pointcloudStream, _ := pointcloud.ReadPCD(bytes.NewReader(pcd))
-	t.Logf("Pointcloud points: %v", pointcloudStream.Size())
-	test.That(t, pointcloudStream.Size(), test.ShouldBeGreaterThanOrEqualTo, 100)
+	pointcloud, _ := pointcloud.ReadPCD(bytes.NewReader(pcd))
```

Comment:
because integration tests use the slam service methods from rdk(`GetPosition`, `GetInternalStateFull`, and `GetPointCloudMapFull`), the functions used here are still valid and will be updated by nicks PR